src/app/common/components/modals/dialog-create-or-update-wallet/dialog-create-or-update-wallet.component.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Lets maybe simplify naming for this component and just name it `WalletModalComponent` or `WalletFormModalComponent`